File: EstructurasControl/Condicionales/Anidadas/Csharp/ejercicio1.cs
using System;

class ProgramaAnidado1
{
    static void Main()
    {
        Console.Write("Ingresa un número: ");
        int numero = Convert.ToInt32(Console.ReadLine());

        // Condicional anidada
        if (numero >= 0)
        {
            if (numero == 0)
            {
                Console.WriteLine("El número es cero.");
            }
            else
            {
                Console.WriteLine("El número es positivo.");
            }
        }
        else
        {
            Console.WriteLine("El número es negativo.");
        }
    }
}

File: EstructurasControl/Condicionales/Anidadas/Python/ejercicio1.py
print("=== Verificación de número ===")

numero = int(input("Ingresa un número: "))

# Condicional anidada
if numero >= 0:
    if numero == 0:
        print("El número es cero.")
    else:
        print("El número es positivo.")
else:
    print("El número es negativo.")
